src/components/big-button/big-button.css

```
@@ -1,4 +1,17 @@
 .BigButton {
-    border: 1px solid white;
+    height: 105px;
+    max-width: 312px;
```

max-width will be 100% for small screens. Replace max-width by width and set a max-width of 100%
```suggestion
    width: 312px;
    max-width: 100%;
```